modules/end-user-guide/partials/ref_parameters-for-che-editor-yaml.adoc

```diff
@@ -1,17 +1,27 @@
 [id="parameters-for-che-editor-yaml"]
 = Parameters for che-editor.yaml
 
-The simplest way to select an IDE in the `che-editor.yaml` is to specify the `id` of an IDE that is available in the table of supported in-browser IDEs in xref:selecting-a-workspace-ide.adoc[]:
+The simplest way to select a cloud IDE in the `che-editor.yaml` is to specify the `id` of an IDE from the table of supported cloud IDEs:
```

Comment:
just curious why we keep saying "cloud IDE" instead of just "IDE" or "editor" ? 

pycharm isn't a cloud IDE as such, as it runs offline too. But it's one of the options in Che (not in Dev Spaces).

Reply:
I had three reasons for using the "cloud IDE" term:
1. I based it on your [response](https://issues.redhat.com/browse/CRW-3098?focusedCommentId=21568936&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-21568936).

2. Without insulting the target audience's intelligence, a user who has a locally installed IDE on their machine can find any mentions of "IDE" (as opposed to "cloud IDE") confusing or misleading.

3. > One could say the same about IDEA and VSCode... they work offline too, so they're not strictly CLOUD IDEs.
-- You mean I start a workspace and the workspace IDE that I see in the browser tab can "work offline"?

Reply:
Re 3:
I mean you can install intelliJ IDEA or vscode / codium on your own computer .. OR run it in a browser connected to a cluster.

Same basic functionality, but one is desktop IDE, and the other is cloud IDE.

Reply:
@nickboldt, sure, so if we're not discussing a `desktop IDE` in the docs, can't we narrow it down to `cloud IDE` to make it clear which implementation of that IDE the docs mean?

Reply:
> I'm pretty sure a user will know if they are using something they installed on their computer or that they're using from within their browser / the cloud (aka someone else's computer).

That's beyond doubt, what's not beyond doubt to me is seeing an IDE mentioned on a docs page.

Reply:
Another aspect to consider is marketing and SEO:
https://codegeekz.com/best-cloud-based-ides/
https://codeanywhere.com/
https://www.slant.co/topics/713/~best-cloud-ides
This could lead to more customers for Che/Dev Spaces.

Reply:
BTW, I'm fine with either option: `IDE` or `cloud IDE`, just to stick with one from now on.

Reply:
I am fine with both too. I don't think we need to specify "cloud", but if @max-cx thinks it's important let's do that.

Reply:
Thank you. I've consulted with our docs team, and we have consensus on using `IDE`, although some of us (including myself) are open to using `cloud IDE`. So we'll stick to `IDE` unless needed.